docs/admin/extensible-admission-controllers.md

```diff
@@ -0,0 +1,152 @@
+
+* uninitialized objects are not visible ...
+
+### When are initializers called?
+
+## External Admission Webhooks (assigned to @whitlockjc)
+
+###What are they? (assigned to @whitlockjc)
+External webhook: non-mutating, called in parallel
+
+### When are they called? (assigned to @whitlockjc)
+
+AdmissionChain, depends on the admission-controller-config
+
+Recommended plug-in order:
+???
+```
+--admission-control=Initializer,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds,GenericAdmissionWebhook"
+```
+
+## How to enable initializers and external admission webhooks?
+
+Initializers and External Admission Webhooks are alpha features, which are disabled by default. To turn them on, you need to
+
+* include "Initializer" and "GenericAdmissionWebhook" in the `--admission-control` flag when starting the apiserver. If you are using HA-master or aggregated apiservers, you need to do so for each apiserver.
+
+* enable the dynamic admission controller registration API by setting `--runtime-config=admissionregistration.k8s.io/v1alpha1` when starts the kube-apiserver.
+
+## How to write an initializer (@Clayton)
+## How to deploy an initializer (@Clayton)
+## How to configure initializers on the fly?
+
+You can configure what initializers are enabled and what resources are subject to the initializers via creating initiallizerconfigurations.
+
+We suggest that you first deploy the initializer controller and make sure it is working properly before creating the initiallizerconfigurations, otherwise the resources will stuck in the uninitialized state.
+
+The following is an example initiallizerconfigurations.
+
+```yaml
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: InitializerConfiguration
+metadata:
+  name: example-config
+spec:
+  initializers:
+    # the name needs to be fully qualified, i.e., containing two "."
+    - name: podimage.example.com
+      rules:
+        # apiGroups, apiVersion, resources all support wildcard "*".
+        # "*" cannot be mixed with non-wildcard.
+        - apiGroups:
+            - ""
+          apiVersions:
+            - v1
+          resources:
+            - pods
+```
+
+For a Create request received by the apiserver, if the request matches any of the `rules` of an initializer, the `Initializer` admission controller will add the initializer to the `metadate.initializers` field of the created object, thus the initializer controller will notice the creation and initialize the object.
+
+It is recommended to make sure that all expansions of the `<apiGroup, apiVersions,resources>` tuple in a `rule` are valid; if they are not, separate them to different `rules`.
+
+After you create the initializerConfiguration, please give the system a few seconds to honor the new configuration.
+
+## How to write a webhook admission controller?
```

Comment:
Consolidate all the Initializer stuff together and all of the Webhook Admission Controller stuff together. 

e.g. (1) what is it?/when to use it? (2) when is it called? (3) how to write it? (4) how to deploy it? (5) how to configure it? These should all be in one section, rather than interwoven (as they are currently).